libnavigator/api/current.txt

```diff
@@ -0,0 +1,14 @@
+// Signature format: 3.0
+package com.mapbox.navigation.navigator {
+
+  public final class LocationExKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions

Reply:
looks like everything in libnavigator is "internal"

Reply:
That's right and it needs to! Let's track it over time 🚀